cephprimarystorage/cephprimarystorage/cephdriver.py

```
@@ -40,7 +40,9 @@ def create_volume(self, cmd, rsp, agent=None):
         if cmd.skipIfExisting and shell.run("rbd info %s" % path) == 0:
             return rsp
 
-        if ceph.is_xsky():
+        if shell.run('rbd info %s' % path) == 0:
```

Ruff (0.12.2)

`shell` may be undefined, or defined from star imports

(F405)